src/modules/datepicker/classes/calendar-mappings.ts

```diff
@@ -69,7 +69,7 @@ export class DatetimeMappings extends CalendarMappings {
             [CalendarViewType.Month, CalendarViewType.Year],
             [CalendarViewType.Date, CalendarViewType.Month],
             [CalendarViewType.Hour, CalendarViewType.Date],
-            [CalendarViewType.Minute, CalendarViewType.Date]
+            [CalendarViewType.Minute, CalendarViewType.Hour]
```

Comment:
I'll wait for this to be decided on to do the PR.

Points in favor of the proposed change:
1. "zoomOut" shouldn't skip a step
2. If user wants to change hour, will be forced to re-select date

Points against the proposed change:
1. After date is selected, to clicks are needed to get to the date view